Examples/Algorithms/Geant4/src/ParticleTrackingAction.cpp

```diff
@@ -102,15 +102,15 @@ void ActsExamples::ParticleTrackingAction::PostUserTrackingAction(
 ActsExamples::SimParticle ActsExamples::ParticleTrackingAction::convert(
     const G4Track& aTrack, SimBarcode particleId) const {
   // Unit conversions G4->::ACTS
-  constexpr double convertTime = Acts::UnitConstants::s / CLHEP::s;
```

Comment:
Changing this from s to ns shouldn't make any difference here right ?

Reply:
correct I just aligned that with the other files and I think `ns` is more natural in our context

Reply:
I fully agree with the change, I just wanted to be sure ! The rest seem fine by me. So let me just approve !